src/utils/ioModule.js

```diff
@@ -0,0 +1,20 @@
+import { Console } from '@woowacourse/mission-utils';
+
+const getUserInput = async function getUserInputFunc(promptMessage) {
+  const userInput = await Console.readLineAsync(`${promptMessage}\n`);
+  return userInput;
+};
```

Comment:
저는 개인적으로 템플릿 리터럴 문법을 사용할 때 중간에 어떤 문자열이 추가되는 경우에 사용하는데 희성님은 어떤 기준으로 사용하시나요??
- 제가 템플릿 리터럴 문법을 사용하는 기준
```js
const a = "안녕";
const b = "홍길동";
console.log(`${a} ${b}`); // 중간에 공백이 있어서 템플릿 리터럴 문법 사용
console.log(`${a}-${b}`); // 중간에 다른 문자열이 있어서 템플릿 리터럴 문법 사용
console.log(a + b); // + 연산자로만 해결할 수 있다면 + 연산자 사용
```
```suggestion
const getUserInput = function getUserInputFunc(promptMessage) {
  return Console.readLineAsync(promptMessage + "\n");
};
```
그리고 별도의 로직이 없기 때문에 이렇게 `async/await` 키워드를 생략하고 바로 `return`해도 문제 없을 것 같아요!

Reply:
airbnb js 컨벤션에서 문자열을 생성하는 경우 문자열 연결 대신 템플릿 문자열을 사용하라는 내용이 있습니다. 따라서 저는 값을 문자열을 만들어야 하는 상황에서 모두 템플릿 문자열을 사용하고 있습니다! 해당 내용 링크 함께 첨부해 드리겠습니다
https://github.com/airbnb/javascript?tab=readme-ov-file#es6-template-literals

Reply:
getUserInput에서 userInput을 반환한 이유는 반환된 값이 파이프라인을 통해 유효성 검사 등의 후속 처리를 받기 위함입니다! pipeline 함수는 함수 배열을 순차적으로 실행하며 각 함수의 결과를 다음 함수의 입력으로 전달하는 구조이기 때문에 userInput을 반환하여 다음 유효성 검사 단계를 수행할 수 있도록 했습니다.

Reply:
> airbnb js 컨벤션에서 문자열을 생성하는 경우 문자열 연결 대신 템플릿 문자열을 사용하라는 내용이 있습니다. 따라서 저는 값을 문자열을 만들어야 하는 상황에서 모두 템플릿 문자열을 사용하고 있습니다! 해당 내용 링크 함께 첨부해 드리겠습니다 https://github.com/airbnb/javascript?tab=readme-ov-file#es6-template-literals

요 내용은 제 PR에서 다른 분께서도 같은 리뷰를 해주셔서 제가 답변을 달았는데 그 링크를 첨부해놓겠습니다🙃
https://github.com/woowacourse-precourse/javascript-racingcar-7/pull/54#discussion_r1820697852

Reply:
> getUserInput에서 userInput을 반환한 이유는 반환된 값이 파이프라인을 통해 유효성 검사 등의 후속 처리를 받기 위함입니다! pipeline 함수는 함수 배열을 순차적으로 실행하며 각 함수의 결과를 다음 함수의 입력으로 전달하는 구조이기 때문에 userInput을 반환하여 다음 유효성 검사 단계를 수행할 수 있도록 했습니다.

이 부분은 `getUserInput` 함수 내부에 추가적인 로직이 없기 때문에 `async/await` 키워드를 사용하지 않더라도 똑같은 동작을 해서 외부에서 사용할 때만 `await` 키워드를 사용해주면 되기 때문에 `getUserInput` 함수에서는 `async/await` 키워드가 없어도 될 것 같다는 리뷰였습니다:)